src/components/EstimatedFee.tsx

```diff
@@ -14,51 +14,104 @@
 // You should have received a copy of the GNU General Public License
 // along with Parity Bridges UI.  If not, see <http://www.gnu.org/licenses/>.
 
-import { useEffect, useState } from 'react';
-import { Typography, makeStyles } from '@material-ui/core';
 import React from 'react';
+import { Typography, Tooltip } from '@material-ui/core';
+import { fade, makeStyles } from '@material-ui/core/styles';
+import HelpOutlineIcon from '@material-ui/icons/HelpOutline';
 import { useSourceTarget } from '../contexts/SourceTargetContextProvider';
 import { useTransactionContext } from '../contexts/TransactionContext';
 import { transformToBaseUnit } from '../util/evalUnits';
+import round from 'lodash/round';
 import { Alert } from '.';
 
-const useStyles = makeStyles(() => ({
+const useStyles = makeStyles((theme) => ({
   container: {
-    minHeight: '20px'
+    minHeight: '20px',
+    display: 'flex'
+  },
+  tooltipIcon: {
+    ...theme.typography.body1,
+    marginTop: 2,
+    marginLeft: 2,
+    '&:not(:hover)': {
+      color: fade(theme.palette.text.hint, 0.75)
+    }
   }
 }));
 
+const getAmount = (fee: string | null, chainDecimals: number) =>
+  fee ? round(parseFloat(transformToBaseUnit(fee, chainDecimals)), 2) : null;
```

Comment:
Wouldn't it be better to have it formatted by `polkadot.js/api` function with all of the SI units?

Reply:
I think it looks much better now:

![Screen Shot 2021-10-21 at 10 58 01](https://user-images.githubusercontent.com/12089572/138293057-03767742-4c70-49fb-a65b-03d8d222d49e.png)